applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_sink.c

```diff
@@ -701,6 +744,35 @@ int broadcast_sink_enable(le_audio_receive_cb recv_cb)
 		return ret;
 	}
 
+	if (IS_ENABLED(CONFIG_BT_CSIP_SET_MEMBER_TEST_SAMPLE_DATA)) {
+		LOG_WRN("CSIP test sample data is used, must be changed "
+			"before production");
+	} else {
+		if (strcmp(CONFIG_BT_SET_IDENTITY_RESOLVING_KEY_DEFAULT,
+			   CONFIG_BT_SET_IDENTITY_RESOLVING_KEY) == 0) {
+			LOG_WRN("CSIP using the default SIRK, must be changed "
+				"before production");
+		}
+		memcpy(csip_param.sirk, CONFIG_BT_SET_IDENTITY_RESOLVING_KEY, BT_CSIP_SIRK_SIZE);
+	}
+
+	ret = bt_cap_acceptor_register(&csip_param, &csip);
+	if (ret) {
+		LOG_ERR("Failed to register CAP acceptor. Err: %d", ret);
+		return ret;
+	}
+
+	ret = bt_csip_set_member_generate_rsi(csip, csip_rsi_adv_data);
+	if (ret) {
+		LOG_ERR("Failed to generate RSI. Err: %d", ret);
+		return ret;
+	}
+
+	if (IS_ENABLED(CONFIG_BT_AUDIO_SCAN_DELEGATOR)) {
+		ret = bt_mgmt_adv_start(0, ad, ARRAY_SIZE(ad), NULL, 0, true);
+		ERR_CHK(ret);
+	}
```

Comment:
This shouldn't be started from here, we should have something similar to what we do in unicast server where we have a unicast_server_adv_populate function and then the starting of the advertising is done from main.